cmd/shield/main.go

```diff
@@ -127,7 +134,7 @@ func main() {
 	    ######     ##    ##     ##    ##     #######   ######
 	*/
 
-	c.Dispatch("status", "Query the SHIELD backup server for its status and version info",
+	c.Dispatch("status", "Query the SHIELD backup server for its status and version info\n",
```

Comment:
This and other newlines seem to have been added to force blank lines between groups of things?  If so, we should modify HelpBreak() to output an additional newline, rather than relying on the string-pasting implementation currently at play.